aptos-move/aptos-vm/src/transaction_validation.rs

```diff
@@ -196,7 +197,11 @@ pub(crate) fn run_multisig_prologue(
         bcs::to_bytes(&payload).map_err(|_| unreachable_error.clone())?
     } else {
         // Default to empty bytes if payload is not provided.
-        bcs::to_bytes::<Vec<u8>>(&vec![]).map_err(|_| unreachable_error)?
+        if features.is_multisig_v2_fix_enabled() {
+            vec![]
+        } else {
+            bcs::to_bytes::<Vec<u8>>(&vec![]).map_err(|_| unreachable_error)?
```

Comment:
Can this ever happen? Can't we just return `vec![]`?

Reply:
When a multisig transaction does not have a provided payload, we currently use that value to represent it.